File: backend/src/main/java/com/example/backend/domain/vote/dto/MostVotedLocationDto.java
package com.example.backend.domain.vote.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class MostVotedLocationDto {
    private String address;
    private Double latitude;
    private Double longitude;
}

File: backend/src/main/java/com/example/backend/domain/vote/dto/MostVotedLocationDto.kt
package com.example.backend.domain.vote.dto

import com.fasterxml.jackson.annotation.JsonProperty

data class MostVotedLocationDto(
    @JsonProperty("location") val location: String,
    @JsonProperty("address") val address: String,
    @JsonProperty("latitude") val latitude: Double,
    @JsonProperty("longitude") val longitude: Double
)